server/jetstream_test.go

```diff
@@ -3015,6 +3015,32 @@ func TestJetStreamConsumerAckAck(t *testing.T) {
 	testAck(AckNak)
 	testAck(AckProgress)
 	testAck(AckTerm)
+
+	// checking replies and errors on two explicit acks for the same message or out of range ack
+	// first get the message
+	m, err := nc.Request(rqn, nil, 10*time.Millisecond)
+	require_NoError(t, err)
+
+	var resp JSApiConsumerAckResponse
+	// Send a request for the first ack and make sure it worked.
+	ackReply1, err := nc.Request(m.Reply, AckAck, 10*time.Millisecond)
+	require_NoError(t, err)
+	require_NoError(t, json.Unmarshal(ackReply1.Data, &resp))
+	require_True(t, resp.Success)
+
+	// Send a second ack and make sure it fails.
+	ackReply2, err := nc.Request(m.Reply, AckAck, 10*time.Millisecond)
+	require_NoError(t, err)
+	resp = JSApiConsumerAckResponse{}
+	require_NoError(t, json.Unmarshal(ackReply2.Data, &resp))
+	require_True(t, !resp.Success && resp.Error.ErrCode == uint16(JSConsumerMsgNotPendingAckErr))
```

Comment:
If this `require_True` (and also the last one) fail you only see "was `false`" and have no context.
Could these be split into two, something like the following so you have more explicit error messages?
```go
require_False(t, resp.Success)
require_Equal(t, resp.Error.ErrCode, uint16(JSConsumerMsgNotPendingAckErr))
```

Reply:
Sure, done.